doc/install.rst

```diff
@@ -28,6 +28,51 @@ There are different ways to install scikit-learn:
 Installing the latest release
 =============================
 
+Quickstart
+----------
+
+.. raw:: html
+
+  <div class="install">
+       <p><strong>System</strong></p>
+          <input type="radio" name="os" id="quickstart-mac" checked>
+          <label for="quickstart-mac">macOS / OSX</label>
+          <input type="radio" name="os" id="quickstart-win">
+          <label for="quickstart-win">Windows</label>
+          <input type="radio" name="os" id="quickstart-lin">
+          <label for="quickstart-lin">Linux</label>
+       <p><strong>Packager</strong></p>
+          <input type="radio" name="packager" id="quickstart-pip" checked>
+          <label for="quickstart-pip">pip</label>
+          <input type="radio" name="packager" id="quickstart-conda">
+          <label for="quickstart-conda">conda</label>
+          <input type="radio" name="packager" id="quickstart-source">
+          <label for="quickstart-source">source</label>
+       <p><strong>Pip Virtual Environment</strong></p>
+          <input type="checkbox" name="config" id="quickstart-venv">
+          <label for="quickstart-venv">Virtual environment</label>
+
+       <div><pre>
+       <code>
+          <span class="sk-expandable linux-venv">python3 -m venv .env</span>
+          <span class="sk-expandable venv">python -m venv .env</span>
+          <span class="sk-expandable venv">source .env/bin/activate</span>
+          <span class="sk-expandable win-venv">.env\Scripts\activate</span>
+          <span class="sk-expandable pip">pip install -U scikit-learn</span>
+          <span class="sk-expandable conda">conda install scikit-learn </span>
+          <span class="sk-expandable source">git clone https://github.com/scikit-learn/scikit-learn</span>
+          <span class="sk-expandable source">cd scikit-learn</span>
+          <span class="sk-expandable linux-source">export PYTHONPATH=`pwd`</span>
+          <span class="sk-expandable win-source">set PYTHONPATH=/path/to/scikit-learn</span>
+          <span class="sk-expandable source">pip install -r requirements.txt</span>
+          <span class="sk-expandable source">python install --editable .</span>
+       </code>
+       </pre></div>
+  </div>
+
```

Comment:
I would add:

Check your installation:

[when "packager" is "pip" or "source"]
```
python -m pip show scikit-learn  # to see which version and where scikit-learn is installed
python -m pip freeze  # to see all packages installed in the active virtualenv
```

[when "package" is "conda"]
```
conda list scikit-learn  # to see which scikit-learn version is installed
conda list  # to see all packages installed in the active conda environment
```

Using an isolated environment (virtualenv or conda environment) makes it possible to install a specific version of scikit-learn and its dependencies independently of any previously installed Python packages. In particular under Linux is it discouraged to install pip packages alongside the packages managed by the package manager of the distribution (apt, dnf, pacman...).

Note that you should always remember to activate the environment of your choice prior to running any Python command whenever you start a new terminal session.

Reply:
Addressed in the latest commits.